aptos-move/framework/aptos-experimental/sources/trading/market/market_types.move

```diff
@@ -126,13 +128,16 @@ module aptos_experimental::market_types {
         cleanup_order_f: |address, OrderIdType, bool, u64| has drop + copy,
         /// decrease_order_size_f arguments: account, order_id, is_bid, price, size
         decrease_order_size_f: |address, OrderIdType, bool, u64, u64| has drop + copy,
+        /// get a string representation of order metadata to be used in events
+        get_order_metadata_string: |M| String has drop + copy
```

Comment:
should we do this, or should we bcs serialize OrderMetadata object? 

@BriungRi  for thoughts as well?

Reply:
One advantage of string is that we don't need to put entire metadata in the event itself but we can change it to bcs serialized as well if that's preferred.

Reply:
cc - @dermanyang

Reply:
maybe it should be a vec<u8> then - so that you can choose which type to serialize? i.e. you can create a subset type to serialize